modules/ipi-install-creating-an-rhcos-images-cache.adoc

```diff
@@ -107,3 +109,33 @@ $ podman run -d --name rhcos_image_cache \
 -p 8080:8080/tcp \
 registry.centos.org/centos/httpd-24-centos7:latest
 ----
+ifndef::upstream[]
++
+The above command creates a caching webserver with the name `rhcos_image_cache`, which will serve the images for deployment. The first image `${RHCOS_PATH}${RHCOS_QEMU_URI}?sha256=${RHCOS_QEMU_SHA_UNCOMPRESSED}` is the `bootstrapOSImage` and the second image `${RHCOS_PATH}${RHCOS_OPENSTACK_URI}?sha256=${RHCOS_OPENSTACK_SHA_COMPRESSED}` is the `clusterOSImage` in the `install-config.yaml` file.
+endif::[]
+
+. Generate the `bootstrapOSImage` and `clusterOSImage` configuration.
++
+[source,terminal]
+----
+$ export BAREMETAL_IP=$(ip addr show dev baremetal | awk '/inet /{print $2}' | cut -d"/" -f1)
+$ export RHCOS_OPENSTACK_SHA256=$(zcat /home/kni/rhcos_image_cache/${RHCOS_OPENSTACK_URI} | sha256sum | awk '{print $1}')
+$ export RHCOS_QEMU_SHA256=$(zcat /home/kni/rhcos_image_cache/${RHCOS_QEMU_URI} | sha256sum | awk '{print $1}')
+$ export CLUSTER_OS_IMAGE="http://${BAREMETAL_IP}:8080/${RHCOS_OPENSTACK_URI}?sha256=${RHCOS_OPENSTACK_SHA256}"
+$ export BOOTSTRAP_OS_IMAGE="http://${BAREMETAL_IP}:8080/${RHCOS_QEMU_URI}?sha256=${RHCOS_QEMU_SHA256}"
+$ echo "${RHCOS_OPENSTACK_SHA256}  ${RHCOS_OPENSTACK_URI}" > /home/kni/rhcos_image_cache/rhcos-ootpa-latest.qcow2.md5sum
+$ echo "    bootstrapOSImage=${BOOTSTRAP_OS_IMAGE}"
+$ echo "    clusterOSImage=${CLUSTER_OS_IMAGE}"
+----
```

Comment:
Can you split these apart so that they're individually usable via copy/paste button?